tests/optimum_cli_export.rs

```diff
@@ -146,7 +146,7 @@ create_test! {
     repo_name: "all-mpnet-base-v2",
     repo_owner: "sentence-transformers",
     name: optimum_cli_export_all_mpnet_base_v2,
-    pooling: Some(Pooling::Cls),
+    pooling: Some(Pooling::Mean),
```

Comment:
This did not affect the `expected_embedding_dim` values?

Reply:
They don't, because this model exports `Array2<f32>` instead of `Array3<f32>` as others would. So the new check at the top of `pooling::mean` will just short circuit.

It would be nice if we can read the `config.json` or something to decide what to do with output keys and the post-processing, instead of relying blindly on the observed dimension of the output array; but I can't see anything in those configuration files that defines things one way or another.

Reply:
I should probably make 2 more tests to check that `Pooling::Cls` still works.

Reply:
Thanks for the clarification.

Reply:
Added the tests now.